README.md

```diff
@@ -594,6 +594,16 @@ This is the **only** place in Puppet where backslash separators should be used.
 Note that backslashes in double-quoted strings *must* be double-escaped and
 backslashes in single-quoted strings *may* be double-escaped.
 
+> **Note on Secrets in `install_options`
+>
+> There is no guarantee that secrets in `install_options` will not show up in debug runs of either `puppet agent` or `puppet apply` calls.
+> This is another reason to _not_ set your production runs to debug mode.
+> However, this information is not written to puppetdb or any other Puppet logs.
+>
+> It **is** written to the Chocolatey log on each machine unless you have C4B and use the `--package-parameters-sensitive` flag, which will redact these values from the Chocolatey log.
+>
+> If you need to include a secret in your `install_options`, do not run in debug mode in production and use C4B and the `--package-parameters-sensitive` flag.
```

Comment:
There is also `--install-arguments-sensitive`

Reply:
If you want to link to more information, that is at https://chocolatey.org/docs/commands-install#options-and-switches and https://chocolatey.org/docs/commands-upgrade#options-and-switches

Reply:
Updated to include this flag and links to the docs, thanks!